modules/dxp/apps/portal-security-audit/portal-security-audit-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -33,47 +32,12 @@ page import="com.liferay.portal.security.audit.web.internal.display.context.Audi
 
 <%@ page import="java.text.Format" %>
 
-<%@ page import="java.util.Calendar" %>
-
 <liferay-frontend:defineObjects />
 
 <liferay-theme:defineObjects />
 
 <portlet:defineObjects />
 
 <%
-Calendar today = CalendarFactoryUtil.getCalendar(timeZone, locale);
-
-today.add(Calendar.MINUTE, 1);
-
-Calendar yesterday = CalendarFactoryUtil.getCalendar(timeZone, locale);
-
-yesterday.add(Calendar.DATE, -1);
-
-String className = ParamUtil.getString(request, "className");
-String classPK = ParamUtil.getString(request, "classPK");
-String clientHost = ParamUtil.getString(request, "clientHost");
-String clientIP = ParamUtil.getString(request, "clientIP");
-String eventType = ParamUtil.getString(request, "eventType");
-String serverName = ParamUtil.getString(request, "serverName");
-int serverPort = ParamUtil.getInteger(request, "serverPort");
-long groupId = ParamUtil.getLong(request, "groupId");
-long userId = ParamUtil.getLong(request, "userId");
-String userName = ParamUtil.getString(request, "userName");
-
-int endDateAmPm = ParamUtil.getInteger(themeDisplay.getRequest(), "endDateAmPm", today.get(Calendar.AM_PM));
-int endDateDay = ParamUtil.getInteger(themeDisplay.getRequest(), "endDateDay", today.get(Calendar.DATE));
-int endDateHour = ParamUtil.getInteger(themeDisplay.getRequest(), "endDateHour", today.get(Calendar.HOUR));
-int endDateMinute = ParamUtil.getInteger(themeDisplay.getRequest(), "endDateMinute", today.get(Calendar.MINUTE));
-int endDateMonth = ParamUtil.getInteger(themeDisplay.getRequest(), "endDateMonth", today.get(Calendar.MONTH));
-int endDateYear = ParamUtil.getInteger(themeDisplay.getRequest(), "endDateYear", today.get(Calendar.YEAR));
-
-int startDateAmPm = ParamUtil.getInteger(themeDisplay.getRequest(), "startDateAmPm", yesterday.get(Calendar.AM_PM));
-int startDateDay = ParamUtil.getInteger(themeDisplay.getRequest(), "startDateDay", yesterday.get(Calendar.DATE));
-int startDateHour = ParamUtil.getInteger(themeDisplay.getRequest(), "startDateHour", yesterday.get(Calendar.HOUR));
-int startDateMinute = ParamUtil.getInteger(themeDisplay.getRequest(), "startDateMinute", yesterday.get(Calendar.MINUTE));
-int startDateMonth = ParamUtil.getInteger(themeDisplay.getRequest(), "startDateMonth", yesterday.get(Calendar.MONTH));
-int startDateYear = ParamUtil.getInteger(themeDisplay.getRequest(), "startDateYear", yesterday.get(Calendar.YEAR));
-
 Format dateFormatDateTime = FastDateFormatFactoryUtil.getDateTime(locale, timeZone);
 %>
```

Comment:
@ChrisKian, because the pr optimizes the init.jsp, please also add one new commit to move the code Format dateFormatDateTime = FastDateFormatFactoryUtil.getDateTime(locale, timeZone); to view_audit_event.jsp. Only view_audit_event.jsp uses the var.